qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/CreateCustomerView.groovy

```diff
@@ -88,9 +90,14 @@ class CreateCustomerView extends VerticalLayout {
         styleDetailsToggle()
 
         this.submitButton = new Button("Create Customer")
-        submitButton.setIcon(VaadinIcons.USER)
+        submitButton.setIcon(VaadinIcons.USER_CHECK)
+        submitButton.addStyleName(ValoTheme.BUTTON_FRIENDLY)
         submitButton.enabled = allValuesValid()
 
+        this.abortButton = new Button("Abort Customer Creation")
+        abortButton.setIcon(VaadinIcons.EXIT)
+        abortButton.addStyleName(ValoTheme.BUTTON_DANGER)
```

Comment:
This is the coloring of the button, right?
I like the idea of pimping the style of the view but in my opinion, this does not fit the style that we have. I would also prefer to use the icon for the button that we already have. (The icon `VaadinIcons.USER`).

Reply:
I reverted the icon but would like to keep the coloring intact for presentations sake. I agree that in later stages we need to refactor the buttons anyhow to include styling options via css